src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -216,11 +217,16 @@ class connectionStrings extends viewModelBase {
             });
     }
     
-    private getTasksThatUseThisString(connectionStringName: string, taskType: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskType) : { taskName: string; taskId: number }[]{        
-        const tasksData = this.connectionStringsTasksInfo[connectionStringName];               
-               
-        return tasksData ? _.sortBy(tasksData.map((task) => { return { taskName: task.TaskName, taskId: task.TaskId }; }), 
-                                    x => x.taskName.toUpperCase()) : [];    
+    private getTasksThatUseThisString(connectionStringName: string, taskType: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskType): { taskName: string; taskId: number }[] {
+        if (!this.connectionStringsTasksInfo[connectionStringName]) {
+            return [];
+        }
```

Comment:
it is still not valid javascript formatting:

if (..) {
   ..
} else {    // this goes in single line
    ...
}

Reply:
done